docs/guide/exe_verilator.rst

```diff
@@ -1270,6 +1270,11 @@ Summary:
 
    Disable tracing of parameters.
 
+.. option:: --trace-strings
```

Comment:
Maybe have this take the number of characters to allow in a string, defaulting to zero?

Might make a mess of you needing a constant currently.

Reply:
Yeah, I'll try this and see how much cleaner/uglier things get.